spectral_cube/stokes_spectral_cube.py

```diff
@@ -45,8 +49,9 @@ def __init__(self, stokes_data, mask=None, meta=None, fill_value=None):
                                  "should have the same WCS")
 
             if component not in VALID_STOKES:
-                raise ValueError("Invalid Stokes component: {0} - should be "
-                                 "one of I, Q, U, V, RR, LL, RL, LR, XX, XY, YX, YY, RX, RY, LX, LY, XR, XL, YR, YL, PP, PQ, QP, QQ, RCircular, LCircular, Linear, Ptotal, Plinear, PFtotal, PFlinear, Pangle".format(component))
+                raise ValueError("Invalid Stokes component: {0} - should be one of I, Q, U, V, RR, LL, RL, LR, XX, XY, YX, YY, \
+                                 RX, RY, LX, LY, XR, XL, YR, YL, PP, PQ, QP, QQ, \
+                                 RCircular, LCircular, Linear, Ptotal, Plinear, PFtotal, PFlinear, Pangle".format(component))
```

Comment:
for future - we could just populate this from `VALID_STOKES` instead of writing it out